gdal/swig/python/gdal-utils/osgeo_utils/__init__.py

```diff
@@ -0,0 +1,10 @@
+__pacakge_name__ = 'gdal-utils'
+__version__ = '3.3.0.6'
```

Comment:
HOWTO-RELEASE should be updated to reflect the new location of this version number

Reply:
done. I've also updated the twine section for gdal-utils and added `twine check`